homeassistant/components/google_assistant/smart_home.py

```diff
@@ -137,7 +137,11 @@ async def async_devices_query(hass, data, payload):
             continue
 
         entity = GoogleEntity(hass, data.config, state)
-        devices[devid] = entity.query_serialize()
+        try:
+            devices[devid] = entity.query_serialize()
+        except Exception:  # pylint: disable=broad-except
+            _LOGGER.exception("Unexpected error serializing query for %s", state)
+            devices[devid] = devices[devid] = {"online": False}
```

Comment:
You assigned this twice.

Reply:
No it looks correct. If we get an exception it won't have been assigned

Reply:
Wouldn't this be sufficient?

```python
devices[devid] = {"online": False}
```

Reply:
I'm blind. You are correct.

Reply:
```suggestion
            devices[devid] = {"online": False}
```

Reply:
oops. Thanks for paying attention 👍